### ISSUE TITLE
Popup button border has two colors when user moves focus to another app

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#6556]
.NET Core Version: 
- 7.0.0-alpha.1.22069.9

Have you experienced this same bug with .NET Framework?: 
- No (regression from #6021)

**Repro steps:**
1. Create a `Form` with a `Button`
2. Set the following button setting (`FlatStyle = FlatStyle.Popup`)
3. Run the app
4. Move focus to another app

**Actual behavior:**
Button border has 2 colors:
![image](https://user-images.githubusercontent.com/23376742/151329572-96979f24-6167-4db2-a3a1-725785e3251d.png)

**Expected color:**
Button border should be a single color.

**Comment by John-Qiao** [comment]
Verified with .NET SDK 7.0.100-preview.3.22153.11 build, this issue is fixed, Button border has a single color.
![TestResult](https://user-images.githubusercontent.com/45864985/156707694-f08be3dc-562a-4fec-927a-ab9c0f2e36eb.png)
